src/Api/AdminConsole/Controllers/SlackIntegrationController.cs

```diff
@@ -46,38 +71,39 @@ public async Task<IActionResult> RedirectAsync(Guid organizationId)
         return Redirect(redirectUrl);
     }
 
-    [HttpGet("create", Name = nameof(CreateAsync))]
-    public async Task<IActionResult> CreateAsync(Guid organizationId, [FromQuery] string code)
+    [HttpGet("integrations/slack/create", Name = nameof(CreateAsync))]
+    [AllowAnonymous]
+    public async Task<IActionResult> CreateAsync([FromQuery] string code, [FromQuery] string state)
     {
-        if (!await currentContext.OrganizationOwner(organizationId))
+        // Fetch existing Initiated record
+        var integration = await integrationRepository.GetByIdAsync(Guid.Parse(state));
```

Comment:
🎨 Need to try parsing and use other safety mechanisms -- people will know this API exists and could send junk at it.